### DISCUSSION TITLE
conflicting features

### DISCUSSION THREAD

**Discussion by PyroTechniac** [xd009642/tarpaulin#875] [Q&A]
Adding coverage to the crate https://github.com/twilight-rs/twilight, and it has some conflicting features (features that compile the same function name when compiled), and I was wondering how to create coverage for it? Specifically with the https://github.com/actions-rs/tarpaulin action?

**Comment by xd009642** [comment]
So you can add a `tarpaulin.toml` or `.tarpaulin.toml` to your repository root as described in the readme: https://github.com/xd009642/tarpaulin#config-file this lets you specify the args for multiple tarpaulin runs and then they all get ran and the results merged
